package.json

```diff
@@ -61,6 +62,9 @@
     "babel-preset-react": "6.24.1",
     "css-loader": "0.28.7",
     "cssnano": "3.10.0",
+    "deep-freeze": "0.0.1",
```

Comment:
Where is this being used?

Reply:
This is needed for reducer tests. Currently not in use.

Reply:
Please add it when we need it. If not in use, then not needed.